ksp_plugin/pile_up.cpp

```diff
@@ -142,30 +181,27 @@ not_null<std::unique_ptr<PileUp>> PileUp::ReadFromMessage(
   std::unique_ptr<PileUp> pile_up;
   if (is_pre_cesàro) {
     if (is_pre_cartan) {
-      pile_up = std::unique_ptr<PileUp>(
-          new PileUp(std::move(parts),
-                     DefaultPsychohistoryParameters(),
-                     DefaultHistoryParameters(),
-                     DiscreteTrajectory<Barycentric>::ReadFromMessage(
-                         message.history(),
-                         /*forks=*/{}),
-                     /*psychohistory=*/nullptr,
-                     ephemeris,
-                     std::move(deletion_callback)));
+      pile_up = std::unique_ptr<PileUp>(new PileUp(
+          std::move(parts),
+          DefaultPsychohistoryParameters(),
+          DefaultHistoryParameters(),
+          DiscreteTrajectory<Barycentric>::ReadFromMessage(message.history(),
+                                                           /*forks=*/{}),
+          /*psychohistory=*/nullptr,
+          ephemeris,
+          std::move(deletion_callback)));
```

Comment:
I'd rather not reformat code without a reason.  It's not clear to me that the new format is better than the old one.